src/Selector/Xpath/Manipulator.php

```diff
@@ -44,7 +44,7 @@ public function prepend($xpath, $prefix)
         }
 
         // Split any unions into individual expressions.
-        foreach (preg_split(self::UNION_PATTERN, $xpath) as $expression) {
```

Comment:
`self::UNION_PATTERN` constant should be removed too. Or is used in other places?